### ISSUE TITLE
Is ground-truth required during training?

### ISSUE THREAD

**Issue by lpy523** [NVlabs/BundleSDF#162]
I noticed that the loss function is mentioned in the paper in both the _Online Pose Graph Optimization_ and _Neural Object Field_ sections, and my question is whether ground-truth pose data is needed to supervise the model throughout its training?

I'm sorry I don't understand the paper well enough, if someone could answer my question, I would greatly appreciate it!

**Comment by wenbowen123** [comment]
No it's not needed. It's only a loss for optimization, w/o comparing to GT.

**Comment by lpy523** [comment]
Where specifically in the code is the loss function in the _Online Pose Graph Optimization_ section defined and called?
